lib/runner.js

```diff
@@ -156,13 +156,26 @@ Runner.prototype.controlFlow = function() {
  * @private
  */
 Runner.prototype.setupGlobals_ = function(browser_) {
-  // Export protractor to the global namespace to be used in tests.
+  if (this.config_.noGlobals) {
+    // Keep $, $$, element, and by/By under the global protractor namespace
+    protractor.browser = browser_;
```

Comment:
What about always assigning these and just not setting globals if `noGlobals` is true? It would make moving between the two easier (although I guess one could argue that opening the door to mixing and matching is a _bad_ thing)